plugins/module_utils/dellemc_idrac.py

```diff
@@ -45,7 +45,11 @@ def __enter__(self):
         self.sdk.importPath()
         self.handle = self.sdk.get_driver(self.sdk.driver_enum.iDRAC, self.idrac_ip, self.creds, pOptions=self.pOp)
         if self.handle is None:
-            msg = "Could not find device driver for iDRAC with IP Address: {0}".format(self.idrac_ip)
+            msg = "Unable to create a driver to communicate with idrac %s. This most often indicates one of the " \
```

Comment:
@grantcurell - reword this to Unable to communicate with idrac %s. This typically indicates a bad " \

Basically you even remove the create a driver words from your suggestion.

Reply:
Finished